examples/fullstack/backend/lib/functions.dart

```diff
@@ -24,14 +24,17 @@ export 'api_types.dart';
 @CloudFunction()
 GreetingResponse function(GreetingRequest request, RequestContext context) {
   final name = request.name ?? 'World';
-  final response = GreetingResponse(salutation: getSalutation(), name: name);
+  final response =
+      GreetingResponse(salutation: _randomSalutation(), name: name);
   context.logger.info('greetingResponse: ${response.toJson()}');
   return response;
 }
 
-final Random rng = Random(DateTime.now().millisecondsSinceEpoch);
+String _randomSalutation() => salutations[_random.nextInt(salutations.length)];
```

Comment:
Good call. I would also propose updating `analysis_options.yaml`. As it is, the recommendation doesn't show up with `dart analyze .`.

As a side note, we probably want to update the examples/templates so that projects like this that are generated reflect best practices we want to propagate.

Reply:
agreed!